smap/sources/safarScraper.py

```diff
@@ -25,31 +25,37 @@ def get_content(url):
 
 # gets the data of air quality and returns an array of dictionaries with data points 
 def get_air_quality_data(opts):
-	url = 'http://safar.tropmet.res.in/map_data.php?for=current&city_id=2'
-	content = get_content(url)
-	x = ast.literal_eval(content)
-	air_quality_data = []
-	for i in range(0, len(x)):
-		try:
-			air_quality_data_city ={}
-			air_quality_data_city['area'] = x[i]['title']
-			air_quality_data_city['latitude'] = x[i]['lat']
-			air_quality_data_city['longtitude'] = x[i]['lng']
-			area_web_data = BeautifulSoup(x[i]['description'], "lxml")
-			data_list = []
-			for data in area_web_data.find_all('td'):
-				data_list.append(data.getText())
-			air_quality_data_city['time'] = data_list[1]
-			air_quality_data_city['pm10'] = int(data_list[6])
-			air_quality_data_city['pm2.5'] = int(data_list[9])
-			air_quality_data_city['pmno2'] = int(data_list[12])
-			air_quality_data_city['pmco'] = int(data_list[15])
-			air_quality_data_city['pmo3'] = int(data_list[18])
-			air_quality_data.append(air_quality_data_city)
-			# print air_quality_data_city
-		except:
-			print "Unexpected error for ",i," : ", sys.exc_info()[0]
-	return air_quality_data
+	try:
+		url = 'http://safar.tropmet.res.in/map_data.php?for=current&city_id=2'
```

Comment:
Why are there two copies of the safar scraper? Can we standardize on one to avoid maintenance issues later?